integration-tests/tzip16-originate-contracts-with-metadata-on-HTTPS-and-fetch-metadata.spec.ts

```diff
@@ -119,7 +119,7 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
             done();
         });
 
-        test('Fetch the metadata which contains emoji of the contract', 2, async (done: () => void) => {
+        test('Verify that the metadata for the contract which contains emoji of the contract can be fetched', 2, async (done: () => void) => {
```

Comment:
'Verify that the metadata for the contract which contains emoji ~of the contract~ can be fetched

Reply:
Changed to:
'Verify that the metadata for the contract which contains emoji can be fetched.'